src/db/db.c

```diff
@@ -701,7 +701,7 @@ dbReplayWait(Db *const this, const String *const targetLsn, const uint32_t targe
             {
                 // Build the query
                 const String *const query = strNewFmt(
-                    "select (checkpoint_%s > '%s')::bool as targetReached,\n"
+                    "select (checkpoint_%s >= '%s')::bool as targetReached,\n"
```

Comment:
Yes, this is because we are now testing for a checkpoint that was created without pg_backup_start(), which means we can't depend on any records existing after the checkpoint.